src/components/CustomCollection/EditableCollectionTitle.tsx

```diff
@@ -1,91 +1,86 @@
-import React, { useState, useRef, useEffect } from 'react'
-import { Textarea } from '@trussworks/react-uswds'
+import React, { useRef, useEffect } from 'react'
+
+import {
+  Button,
+  ButtonGroup,
+  Form,
+  Label,
+  TextInput,
+} from '@trussworks/react-uswds'
 
 import styles from './CustomCollection.module.scss'
-/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
+
 type PropTypes = {
   collectionId: string
   text: string
   onSave: (s: string) => void
+  onCancel: () => void
   onDelete: () => void
+  isEditing: boolean
 }
+
 export const EditableCollectionTitle = ({
   collectionId,
   text,
+  isEditing,
   onSave,
-  onDelete,
+  onCancel,
 }: PropTypes) => {
-  const [isEditing, setEditing] = useState(text === '')
-  const [currentText, setCurrentText] = useState(text)
-  const inputRef = useRef<HTMLTextAreaElement>(null)
+  const inputRef = useRef<HTMLInputElement>(null)
 
+  const handleCancel = () => {
+    const inputEl = inputRef.current as HTMLInputElement
+    inputEl.value = ''
```

Comment:
since the `isEditing` flag changing causes the input element to re-render, I don't think we need to clear the value manually